### DISCUSSION TITLE
When KIA, auto generated missions for that aircraft type cease for a while?

### DISCUSSION THREAD

**Discussion by GrEaSeLiTeNiN** [Khopa/dcs_liberation#1936] [Q&A] [answered]
Anyone notice this? Say you fly the JF17 and are KIA. Auto generated JF17 missions then cease for a while. Seems strange when there are still many JF17s at the airbase. I would have to manually plan missions if I want to use the JF17 again.

**Comment by Starfire13** [comment, marked as answer]
Apophenia.
